docs/understand_mmcv/io.md

```diff
@@ -92,7 +112,9 @@ d
 e
 ```
 
-Then use `list_from_file` to load the list from a.txt.
++ Load from disk
```

Comment:
![image](https://user-images.githubusercontent.com/40779233/136489435-10136731-6454-482e-a67d-f9afe2979359.png)
The doc structure can be improved, maybe we can considering 4th-level title?

Reply:
great suggestion